config.go

```diff
@@ -348,6 +352,8 @@ type Config struct {
 
 	MaxCommitFeeRateAnchors uint64 `long:"max-commit-fee-rate-anchors" description:"The maximum fee rate in sat/vbyte that will be used for commitments of channels of the anchors type. Must be large enough to ensure transaction propagation"`
 
+	AnchorsCommitConfTarget uint32 `long:"anchors-commit-conf-target" description:"The target number of blocks to be used to estimate the fee to use for anchor commitments"`
```

Comment:
Difficult to express succinctly, but can we add a note clarifying that this is only for commitments/force closes? I can see end user confusion thinking that we'll use this for cooperative close, which is not the case?